src/screens/survey/redux/reducers/questionReducer.js

```diff
@@ -0,0 +1,21 @@
+import * as actionType from '../actions/ActionType';
+
+const initialState = {}
```

Comment:
I would think it's best to store each question as an array. In firebase, the questions will probably be identified by a GUID, not Question 1, Question 2, etc.